app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
You should add type annotation for `cafe` parameter. It is good practice to use type annotations as it makes your code more understandable and helps with debugging.